packages/firebase_database/firebase_database_web/lib/src/interop/database.dart

```diff
@@ -37,8 +38,12 @@ App getApp(String? name) {
 /// If [persistent], it remembers the logging state between page refreshes.
 ///
 /// See: <https://firebase.google.com/docs/reference/js/firebase.database#.enableLogging>.
-void enableLogging([logger, bool persistent = false]) =>
-    database_interop.enableLogging(jsify(logger), persistent);
+void enableLogging(bool enable, [bool persistent = false]) {
+  database_interop.enableLogging(
+    enable ? (message) => debugPrint('@firebase/database: $message') : enable,
```

Comment:
default web logger swallows message content for some reason (assuming it passes the message as a second argument of console.log, where `print` accepts only a single arg)